components/hubspot/sources/new-contact-in-list/new-contact-in-list.js

```diff
@@ -0,0 +1,67 @@
+const hubspot = require("../../hubspot.app.js");
+
+module.exports = {
+  key: "hubspot-new-contact-in-list",
+  name: "New Contact in List",
+  description: "Emits an event for each new contact in a list.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    hubspot,
+    lists: {
+      type: "string[]",
+      label: "Lists",
+      optional: false,
+      async options({ page, prevContext }) {
+        const params = {
+          count: 20,
+          offset: Object.keys(prevContext).length != 0 ? prevContext.offset : 0,
+        };
+        const results = await this.hubspot.getLists(params);
+        const options = results.lists.map((result) => {
+          const label = result.name;
+          return {
+            label,
+            value: JSON.stringify({ label, value: result.listId }),
+          };
+        });
+        let offset = params.offset + params.count;
+        return {
+          options,
+          context: { offset },
+        };
+      },
+    },
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
+    for (let list of this.lists) {
+      list = JSON.parse(list);
+      const params = {
+        count: 20,
```

Comment:
Is there any way to filter this API call by time like you do for the others?

Reply:
I don't think this API call allows for filtering by time. https://legacydocs.hubspot.com/docs/methods/lists/get_list_contacts